package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "word-aligner",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

Comment:
Based on the changes it looks like it adds breaking changes, should we make it version 5? @PhotoNomad0